### ISSUE TITLE
MoVQ question

### ISSUE THREAD

**Issue by JincanDeng** [ai-forever/Kandinsky-2#64]
Thanks for your great jobs! I want to finetuned a MoVQ on my own dataset. Any advice about the loss, learning rate and other hyper-parameters about the finetuning process? Thanks!
